src/main/java/com/java_essential/kopaiev/essential/homework/lesson1/task2/Rectangle.java

```diff
@@ -0,0 +1,26 @@
+package com.java_essential.kopaiev.essential.homework.lesson1.task2;
+
+public class Rectangle {
+    private double side1;
+    private double side2;
+
+    public Rectangle (double side1, double side2)
+    {
+        this.side1 = side1;
+        this.side2 = side2;
+    }
+
+    double areaCalculator (double side1, double side2)
+    {
+        double area = side1 * side2;
+        return  area;
```

Comment:
The variable "area" is not needed, just return the result of the operation(return  side1 * side2).

Reply:
fixed